SuperBuild/External_Swig.cmake

```diff
@@ -68,6 +68,8 @@ if(NOT SWIG_DIR AND NOT Slicer_USE_SYSTEM_${proj})
 "#------------------------------------------------------------------------------
 # Added by '${CMAKE_CURRENT_LIST_FILE}'
 
+mark_as_superbuild(SWIG_EXECUTABLE)
```

Comment:
This shouldn't be added to the `_env_script`, it should instead be added at the end of the file. See [here](https://github.com/Slicer/Slicer/blob/1b7888e90e233729e4beb4941e8b33e622373655/SuperBuild/External_VTK.cmake#L260) for an example

Reply:
This should probably be `Slicer_SWIG_EXECUTABLE`

Reply:
The variable Slicer_SWIG_EXECUTABLE doens't exist at this point, should I
```cmake
set(Slicer_SWIG_EXECUTABLE ${SWIG_EXECUTABLE})
```
?